### PR TITLE
Fix chardet as fix requests

### PR DESCRIPTION
**Pull request by amanda11**
We pin requests[security] but that has a check when running that chardet is < 3.1.0, but chardet is not listed as requirement with a pinned requirement.
Also fix typo in in-requirements.txt comments

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -17,6 +17,9 @@ Added
 
 Changed
 ~~~~~~~~~
+* Pin chardet requirement #5101
```
Comment (armab) [inline CHANGELOG.rst:20]: Can you please also include in the Changelog why this change was needed.
Following that, it might be a better fit for a Fixed section, rather then Changed.

Comment (amanda11) [reply CHANGELOG.rst]: Thanks for feedback - fixed

**Review by armab — APPROVED** [review]
:+1:

**Review by blag — APPROVED** [review]

**Comment by blag** [comment]
Should be fixed once I tweak StackStorm/st2ci#194, merge it in, and deploy it.

**Comment by armab** [comment]
It makes sense to merge it as is, because it'll fix the other 3 OS builds in this round.
Let's just wait for TravisCI to :heavy_check_mark:

**Comment by blag** [comment]
Travis tests passed. I'm going to merge this in even with the U16 end-to-end tests failing since we need to get our CI passing.
